apps/admin-gui/src/app/vos/pages/member-detail-page/member-bans/member-bans.component.ts

```diff
@@ -34,7 +39,14 @@ export class MemberBansComponent implements OnInit {
     this.loading = true;
     this.voService.getVoBanForMember(this.member.id).subscribe({
       next: (ban) => {
-        this.bans = [{ ban: ban, member: null, vo: null }];
+        if (ban) {
+          this.voService.getVoById(this.member.voId).subscribe({
+            next: (vo) => {
+              this.bans = [{ ban: ban, member: this.member as RichMember, vo: vo }];
+            },
+            error: () => (this.loading = false),
+          });
+        }
```

Comment:
This should not be needed. The information about member is already displayed since this is the member detail. Same goes for organization.

Reply:
I was under the impression that the entity storage had all relevant objects stored which is not the case. Turns out that if related object is required for authorization it is declared like (for this case) `vo: {id: member.voId, beanName: 'Vo'}`. The advantage being that authorization for actions is not blocked but for this case of edit inside of table does not change much.

Reply:
ok, will fix like this, thanks